user_test.go

```diff
@@ -0,0 +1,18 @@
+package discordgo
+
+import "testing"
+
+func TestUser(t *testing.T) {
+	t.Parallel()
+
+	t.Run("String()", func(t *testing.T) {
```

Comment:
Sure, done w/ e19d4f4acb53dd048c20bd9676e1f823a6b3b9d6. The t.Parallel call should work with 1.6, I used the `t.Run` for organization not parallelism